File: articles/algorithms-in-public-policy-and-governance/haskell/ProceduralReadiness.hs
module Main where

main :: IO ()
main = do
  let dueProcess = 0.58
      transparency = 0.52
      humanReview = 0.60
      appealReadiness = 0.54
      score = (dueProcess + transparency + humanReview + appealReadiness) / 4.0
  putStrLn ("procedural_readiness_score=" ++ show score)

File: articles/algorithms-in-public-policy-and-governance/prolog/procedural_readiness.pl
procedural_readiness(DueProcess, Transparency, HumanReview, AppealReadiness, Score) :-
    Score is (DueProcess + Transparency + HumanReview + AppealReadiness) / 4.

% Query example:
% ?- procedural_readiness(0.58, 0.52, 0.60, 0.54, Score).
